app/assets/stylesheets/application.scss

```diff
@@ -1,3 +1,12 @@
+// Include all of the GOV.UK Frontend styles. This includes fonts, and individual components.
```

Comment:
This comment doesn't seem to be referring to the code it's directly above. Do we need this comment?